### PR TITLE
[Guild Admin] Implemented admin set_coins command

### PR DESCRIPTION
**Pull request by ShootingKing-AM**

### CODE REVIEW

**Review by Intery — APPROVED** [review]
Looks good :+1:

File: bot/modules/guild_admin/economy/set_coins.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+from ..module import module
+
+POSTGRES_INT_MAX = 2147483647
+
+@module.cmd(
+    "set_coins",
+    group="Guild Admin",
+    desc="Set coins on a member."
+)
+@in_guild()
```
Comment (Intery) [inline bot/modules/guild_admin/economy/set_coins.py:17]: Can you add a `guild_admin` ward here please!

**Review by Intery — CHANGES_REQUESTED** [review]

File: bot/modules/guild_admin/economy/set_coins.py (hunk 1 above)
Comment (Intery) [reply bot/modules/guild_admin/economy/set_coins.py:17]: Then you also don't need the `in_guild` ward because it is dependent.

Comment (ShootingKing-AM) [reply bot/modules/guild_admin/economy/set_coins.py]: meow !

**Review by Intery — APPROVED** [review]

**Comment by Intery** [comment]
Thanks for your contribution!

**Comment by ShootingKing-AM** [comment]
Changes looks ugly because staging is not insync with master - else it was just 2 cute lil commits :(

**Pull request merged**